deployments/types/common.json

```diff
@@ -1,138 +1,159 @@
-			"Consensus": "Consensus",
-			"Seal": "Seal",
-			"PreRuntime": "PreRuntime"
-		}
-	},
-	"--2": "Rialto Types",
-	"RialtoBalance": "u128",
-	"RialtoBlockHash": "H256",
-	"RialtoBlockNumber": "u32",
-	"RialtoHeader": {
-		"parent_Hash": "RialtoBlockHash",
-		"number": "Compact<RialtoBlockNumber>",
-		"state_root": "RialtoBlockHash",
-		"extrinsics_root": "RialtoBlockHash",
-		"digest": "RialtoDigest"
-	},
-	"RialtoDigest": {
-		"logs": "Vec<RialtoDigestItem>"
-	},
-	"RialtoDigestItem": {
-		"_enum": {
-			"Other": "Vec<u8>",
-			"AuthoritiesChange": "Vec<AuthorityId>",
-			"ChangesTrieRoot": "RialtoBlockHash",
-			"SealV0": "SealV0",
-			"Consensus": "Consensus",
-			"Seal": "Seal",
-			"PreRuntime": "PreRuntime"
-		}
-	},
-	"--3": "Common types",
-	"Address": "AccountId",
-	"LookupSource": "AccountId",
-	"AccountSigner": "MultiSigner",
-	"SpecVersion": "u32",
-	"RelayerId": "AccountId",
-	"SourceAccountId": "AccountId",
-	"ImportedHeader": {
-		"header": "BridgedHeader",
-		"requires_justification": "bool",
-		"is_finalized": "bool",
-		"signal_hash": "Option<BridgedBlockHash>"
-	},
-	"AuthoritySet": {
-		"authorities": "AuthorityList",
-		"set_id": "SetId"
-	},
-	"Id": "[u8; 4]",
-	"InstanceId": "Id",
-	"LaneId": "Id",
-	"MessageNonce": "u64",
-	"MessageId": "(Id, u64)",
-	"MessageKey": {
-		"lane_id": "LaneId",
-		"nonce:": "MessageNonce"
-	},
-	"InboundRelayer": "AccountId",
-	"InboundLaneData": {
-		"relayers": "Vec<(MessageNonce, MessageNonce, RelayerId)>",
-		"last_confirmed_nonce": "MessageNonce"
-	},
-	"OutboundLaneData": {
-		"latest_generated_nonce": "MessageNonce",
-		"latest_received_nonce": "MessageNonce",
-		"oldest_unpruned_nonce": "MessageNonce"
-	},
-	"MessageData": {
-		"payload": "MessagePayload",
-		"fee": "Fee"
-	},
-	"MessagePayload": "Vec<u8>",
-	"BridgedOpaqueCall": "Vec<u8>",
-	"OutboundMessageFee": "Fee",
-	"OutboundPayload": {
-		"spec_version": "SpecVersion",
-		"weight": "Weight",
-		"origin": "CallOrigin",
-		"call": "BridgedOpaqueCall"
-	},
-	"CallOrigin": {
-		"_enum": {
-			"SourceRoot": "()",
-			"TargetAccount": "(SourceAccountId, MultiSigner, MultiSignature)",
-			"SourceAccount": "SourceAccountId"
-		}
-	},
-	"MultiSigner": {
-		"_enum": {
-			"Ed25519": "H256",
-			"Sr25519": "H256",
-			"Ecdsa": "[u8;33]"
-		}
-	},
-	"MessagesProofOf": {
-		"bridged_header_hash": "BridgedBlockHash",
-		"storage_proof": "Vec<StorageProofItem>",
-		"lane": "LaneId",
-		"nonces_start": "MessageNonce",
-		"nonces_end": "MessageNonce"
-	},
-	"StorageProofItem": "Vec<u8>",
-	"MessagesDeliveryProofOf": {
-		"bridged_header_hash": "BridgedBlockHash",
-		"storage_proof": "Vec<StorageProofItem>",
-		"lane": "LaneId"
-	},
-	"UnrewardedRelayersState": {
-		"unrewarded_relayer_entries": "MessageNonce",
-		"messages_in_oldest_entry": "MessageNonce",
-		"total_messages": "MessageNonce"
-	},
-	"AncestryProof": "()",
-	"MessageFeeData": {
-		"lane_id": "LaneId",
-		"payload": "OutboundPayload"
-	}
+  "--1": "Millau Types",
```

Comment:
Please re-format the file back according to `.editorconfig` or change the formatting there as well.

Reply:
ouh , i didn't notice that until now, fixing it